package.json

```diff
@@ -20,41 +20,44 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^5.1.7",
+    "ember-cli-babel": "^6.3.0",
     "ember-getowner-polyfill": "^1.1.1"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
-    "chai": "^3.5.0",
+    "chai": "^4.1.0",
     "chalk": "^2.0.1",
-    "ember-cli": "2.12.0",
+    "ember-cli": "~2.14.0",
     "ember-cli-addon-tests": "^0.7.0",
-    "ember-cli-chai": "^0.4.1",
-    "ember-cli-dependency-checker": "^2.0.0",
+    "ember-cli-chai": "^0.4.2",
+    "ember-cli-dependency-checker": "^2.0.1",
     "ember-cli-eslint": "^4.1.0",
-    "ember-cli-fastboot": "^1.0.0-beta.16",
-    "ember-cli-htmlbars": "^2.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.6",
-    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-fastboot": "^1.0.0-rc.5",
+    "ember-cli-htmlbars": "^2.0.2",
+    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
+    "ember-cli-inject-live-reload": "^1.6.1",
     "ember-cli-mocha": "^0.14.4",
-    "ember-cli-shims": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-cli-shims": "^1.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^1.2.0",
```

Comment:
This is added by `ember init` - I'd add that so it doesn't come up again and again on every ember-cli update?